### PR TITLE
Created User repository

### PR DESCRIPTION
**Pull request by alanlamaswolox**
## Summary

A user repository with a method that finds one user by username

## Trello card

[link](https://trello.com/b/1EiY4hHX/training-java-alan-lamas)

### CODE REVIEW

File: src/main/java/wolox/training/repositories/UserRepository.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package wolox.training.repositories;
+
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import wolox.training.models.User;
+
+public interface UserRepository extends JpaRepository<User,Long> {
+    Optional<User> findFirstByUsername(final String username);
```
Comment (Gianni-Nicolas) [inline src/main/java/wolox/training/repositories/UserRepository.java:14]: Add java docs in each query of the user and book repositories.

**Review by Gianni-Nicolas — CHANGES_REQUESTED** [review]
Good job, I marked only one detail

**Review by AleFalcon — APPROVED** [review]
It's okay with comment by Nico

File: src/main/java/wolox/training/repositories/BookRepository.java [hunk 2]
```diff
@@ -3,7 +3,14 @@
 import java.util.Optional;
 import org.springframework.data.jpa.repository.JpaRepository;
 import wolox.training.models.Book;
+import wolox.training.models.User;
 
 public interface BookRepository extends JpaRepository<Book,Long> {
+
+    /**
+     *
```
Comment (Gianni-Nicolas) [inline src/main/java/wolox/training/repositories/BookRepository.java]: One of the most important things about javadocs is the explanation of the method being documented.
That description would be missing, indicating the purpose of the method in each of the documented repositories.

**Review by Gianni-Nicolas — CHANGES_REQUESTED** [review]

**Review by Gianni-Nicolas — APPROVED** [review]

**Pull request merged**